packages/cli/src/entry-point.ts

```diff
@@ -0,0 +1,24 @@
+import { existsSync } from "fs";
+import { parsedCli } from "./parse-command-line";
+import { ConfigInternals } from "./config";
+import { Log } from "./log";
+
+export const getEntryPoint = () => {
+	let file: string | null = parsedCli._[1];
+  if (file) return file;
+
+  file = ConfigInternals.getEntryPoint();
+  if (file) return file;
+
+  if (existsSync("./src/index.tsx")) file = "src/index.tsx";
```

Comment:
Correct me if I'm wrong but using the forward slash here could break on Windows.
Perhaps you need to create each path with something like:
`path.join(process.cwd(), 'src', 'index.tsx')`

Reply:
Ah, nice catch! Though I'm pretty sure that under the hood, `existsSync` at some point calls the `resolve()` function which should just take care of the platform specific paths?
Ref: https://github.com/nodejs/node/blob/main/lib/fs.js#L299

Reply:
Good point! I've replaced it with `path.join("src", "index.tsx")` which should give us a guarantee that it's going to work :)